packages/notices/src/store/reducer.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { reject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import onSubKey from './utils/on-sub-key';
+
+/**
+ * Reducer returning the next notices state. The notices state is an object
+ * where each key is a context, its value an array of notice objects.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+const notices = onSubKey( 'context' )( ( state = [], action ) => {
+	switch ( action.type ) {
+		case 'CREATE_NOTICE':
+			// Avoid duplicates on ID.
+			return [
+				...reject( state, { id: action.notice.id } ),
```

Comment:
Oh nice, I was using `filter( not( ... ) )` - it's a nice shortcut and reads much better 👍